### PR TITLE
feat: Copy Record Text

### LINKED ISSUE #1616: [Feature request] Copy text option
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to copy record text but I am not able to. Sometimes I want to copy text and check out it"s dependency parser on another tool but currently there is no straightforward way to do so.

**Describe the solution you'd like**
A copy option in the ":" next to "View Metadata" would be great.

**Describe alternatives you've considered**
1. Alternative can be a copy icon next to ":" but that won"t be neat as copy is a peripheral activity
2. Inspect result item and copy text from developer tools. (Overkill)

### PR DESCRIPTION
**Pull request by leiyre**
closes #1616

This PR adds the "copy text" action in the extra actions menu, it also centralizes the "copy to clipboard" function and makes some small changes to the text

### CODE REVIEW

**Comment by frascuchon** [comment]
Just a comment, regarding the changelog discussion, this should be typed as `style` instead of `feat`, isn't it @leiyre @Amelie-V ??

File: frontend/components/commons/results/RecordExtraActions.vue [hunk 1]
```diff
@@ -126,6 +113,16 @@ export default {
     close() {
       this.open = false;
     },
+    copyToClipboard() {
+      if (isObject(this.record.inputs)) {
+        const text = Object.keys(this.record.inputs).map((key) => {
+          return `${key}\r\n${this.record.inputs[key]}`;
+        });
```
Comment (frascuchon) [inline frontend/components/commons/results/RecordExtraActions.vue]: For text classification, the copied text does not keep the same layout that is shown in the record:

Screenshot 2022-07-21 at 11 29 00

In the example below, the field names should be UPPERCASED (in my opinion) and every field should include a break line (maybe two) before the previous field (see the `headline` field in the image)

Also, I think we could skip the field name for cases when just one field (text) is found in the record. But it should be discussed

cc: @Amelie-V @dvsrepo @leiyre ???

File: frontend/components/commons/results/RecordExtraActions.vue [hunk 2]
```diff
@@ -126,6 +113,16 @@ export default {
...
+        return this.$copyToClipboard(text);
+      } else {
+        return this.$copyToClipboard(this.record.text);
```
Comment (frascuchon) [inline frontend/components/commons/results/RecordExtraActions.vue]: Suggesting a code refactoring, we could move the text generation logic to each record class, keeping the `copyToClipboard` method simpler. Something like:

```javascript
copyToClipboard() {
   return this.$copyToClipboard(this.record.clipboardText); // or some other representative name 
}
```

Then each Record class should implement its own logic to generate the `clipboardText` property. For example, in the case of text classification, you can include the object parsing and all the stuff.

**Review by frascuchon — CHANGES_REQUESTED** [review]

File: frontend/components/commons/results/RecordExtraActions.vue (hunk 1 above)
Comment (leiyre) [reply frontend/components/commons/results/RecordExtraActions.vue]: I agree with skip the name from the field when there is only one

**Review by frascuchon — APPROVED** [review]
It's ok from my side.

I think @Amelie-V should also take a look in a live dev environment before merge it

**Pull request merged**